### PR TITLE
terraform 0.12 compatible formatting

### PR DESCRIPTION
**Pull request by tillepille**

### CODE REVIEW

File: install-calico.tf [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+    private_key = file(var.ssh_private_key)
   }
 
   provisioner "remote-exec" {
-    inline = <<EOF
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/etcd.yaml
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/rbac.yaml
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/calico.yaml
-    EOF
+    inline = ["kubectl apply -f https://docs.projectcalico.org/v3.8/manifests/calico.yaml"]
```
Comment (solidnerd) [inline install-calico.tf]:
```suggestion
    inline = [ "kubectl apply -f https://docs.projectcalico.org/v3.2/gettingtarted/kubernetes/installation/hosted/etcd.yaml", "kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/rbac.yaml","kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/calico.yaml"]
```

Comment (tillepille) [reply install-calico.tf]: Okay, reverted the change and moved it into #13

File: install-calico.tf [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
   }
 
   provisioner "remote-exec" {
-    inline = <<EOF
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/etcd.yaml
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/rbac.yaml
-    kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/calico.yaml
-    EOF
+    inline = [
+      "kubectl apply -f https://docs.projectcalico.org/v3.2/gettingtarted/kubernetes/installation/hosted/etcd.yaml",
```
Comment (solidnerd) [inline install-calico.tf]:
```suggestion
      "kubectl apply -f https://docs.projectcalico.org/v3.2/getting-started/kubernetes/installation/hosted/etcd.yaml",
```

Comment (tillepille) [reply install-calico.tf]: Don't know how this happened but now its fixed and squashed again..

**Pull request merged**